File: 1598-crawler-log-folder/1598-crawler-log-folder.py
class Solution:
    def minOperations(self, logs: List[str]) -> int:
        curFol=0
        for log in logs:
            if log == '../':
                if curFol != 0:
                    curFol-=1 
            elif log != './':
                curFol+=1

        return curFol
        

File: 1598-crawler-log-folder/1598-crawler-log-folder.rb
# @param {String[]} logs
# @return {Integer}
def min_operations(logs)
    curFol=0
    for log in logs
        if log == '../'
            curFol-=1  if curFol != 0
        elsif log == './'
            # do nothing
        else
            curFol+=1
        end
    end
    return curFol  
end